com.unity.render-pipelines.universal/Editor/AssetPostProcessors/MaterialPostprocessor.cs

```diff
@@ -25,6 +26,24 @@ class MaterialReimporter : Editor
     {
         static bool s_NeedToCheckProjSettingExistence = true;
 
+
+        internal static bool IsURPShader(Shader shader)
```

Comment:
Should be part of ShaderUtils

Reply:
I would agree but in URP ShaderUtils is part of the Runtime portion and most of these function are Editor only (IsShaderGraphAsset for ex)